CHANGELOG.md

```diff
@@ -59,6 +59,7 @@
 - Bump Jackson Core to v2.17.1
 - Add facility reassignment view to `RecentPatientsItem`, `OverduePatientItem` and `PatientSearchResultItem`
 - Show patient reassignment sheet when done/back is clicked in `PatientSummary`
+- Add `ReassignPatientSheet`
```

Comment:
Since this is the opening point for this feature. Let's add the CHANGELOG entry in `Features` section? Something like

- `Add warning to show patients eligible for reassignment to CC in Bangladesh`?